common/p_boomfspec.cpp

```diff
@@ -3306,8 +3306,8 @@ bool P_ShootCompatibleSpecialLine(AActor* thing, line_t* line)
 			if (line->flags & ML_SECRET) // they can't open secret doors either
 				return false;
 		}
-		if (!line->id) // e6y //jff 3/2/98 all gun generalized types require tag
-			return false;
+		// if (!line->id) // e6y //jff 3/2/98 all gun generalized types require tag
```

Comment:
Can these just be removed rather than commented out?

Reply:
I commented it out to match all the other commented out ones. Wasn't sure if there was a reason for them being like that.

Reply:
Throughout this file there's a lot of nearly identical checks, some commented out, some not. So it seems like a larger cleanup of this is needed.

Reply:
I've removed it and the other commented out checks, as well as the remaining uncommented checks in the same function.